analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -601,7 +601,7 @@ class Pub(
     override fun getVersion(workingDir: File?): String {
         val result = ProcessCapture(workingDir, command(workingDir), getVersionArguments()).requireSuccess()
 
-        return transformVersion(result.stderr)
+        return transformVersion(result.stdout)
```

Comment:
Commit message: Missing final backtick after "requirements".

Out of curiosity, if this was wrong, how could the version requirement ever have passed?

Reply:
> Out of curiosity, if this was wrong, how could the version requirement ever have passed?

I have no idea, when I implemented this check I was also confused why the version was printed on stderr. My best guess is, that this changed with some Flutter / Dart version.